### PR TITLE
 Translate 2020-10-02-ruby-2-7-2-released.md (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Origin post: https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2020-10-02-ruby-2-7-2-released.md
Actual diff: 5fbf500e

### CODE REVIEW

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+lang: ko
 ---
 
-Ruby 2.7.2 has been released.
+루비 2.7.2이 릴리스되었습니다.
 
-This release contains intentional incompatibility. Deprecation warnings are off by default on 2.7.2 and later.
-You can turn on deprecation warnings by specifying the -w or -W:deprecated option at the command-line.
-Please check the topics below for details.
+이 릴리스는 의도적인 비호환이 있습니다. 2.7.2 이후로 기본 설정에서는 폐기 경고를 출력하지 않습니다.
```
Comment (yous) [inline ko/news/_posts/2020-10-02-ruby-2-7-2-released.md:12]: By searching on GitHub, we have "폐기 예정", "폐기", and "제거 예정". I used "제거 예정" on [this post](https://github.com/ruby/www.ruby-lang.org/blob/master/ko/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md), so we can also change them for consistency.

```suggestion
이 릴리스는 의도적인 비호환이 있습니다. 2.7.2 이후로 기본 설정에서는 폐기 예정 경고를 출력하지 않습니다.
```

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+폐기 경고를 활성화하려면 실행시에 `-w`나 `-W:deprecated`를 넘기세요.
```
Comment (yous) [inline ko/news/_posts/2020-10-02-ruby-2-7-2-released.md:13]:
```suggestion
폐기 예정 경고를 활성화하려면 실행시에 `-w`나 `-W:deprecated`를 넘기세요.
```

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
 author: "nagachika"
-translator:
+translator: "shia"
 date: 2020-10-02 11:00:00 +0000
-lang: en
+lang: ko
 ---
 
-Ruby 2.7.2 has been released.
+루비 2.7.2이 릴리스되었습니다.
```
Comment (yous) [inline ko/news/_posts/2020-10-02-ruby-2-7-2-released.md:10]:
```suggestion
루비 2.7.2가 릴리스되었습니다.
```

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2020-10-02-ruby-2-7-2-released.md]: fixed d3ee4de8ef4b5d4ecfd85d68e978a

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md (hunk 3 above)
Comment (riseshia) [reply ko/news/_posts/2020-10-02-ruby-2-7-2-released.md]: fixed df5e8cfe

File: ko/news/_posts/2020-10-02-ruby-2-7-2-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2020-10-02-ruby-2-7-2-released.md]: fixed d3ee4de8ef4b5d4ecfd85d68e978a

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!

**Pull request merged**